### DISCUSSION TITLE
I am not receiving device verification mail for my github account

### DISCUSSION THREAD

**Discussion by KarthickBalaji18** [community/community#119418] [Accessibility] [answered]
### Verification Mail not received immediately

Hi, I'm trying to log in to my github account. I have provided my credentials, I takes me to the device verification page and asks for the verification code. But, I'm not receiving any verification mails immediately. After a couple of hours or so, I'm getting the verification mail by then it got expired. I'm stuck and not able to login. I just created this new account to post this issue.

**Comment by ItzNiusha** [comment]
Same here.

**Comment by pmatty2** [comment]
Same here had to create a new account just to post this

**Comment by cozyGalvinism** [comment]
Facing the same issue with our CI account, wanted to set up 2FA on it after I received the reminder to turn it on, but I'm not even getting the verification code. And I can't report it either on https://support.github.com/contact/cannot_sign_in because selecting "Other" (since not receiving the verification code is not on the list) causes it to try and send another verification message, which obviously never arrives. That's great design!

**Comment by sawamatsu** [comment]
I am also having the same problem with one of my account.
I received the device verification email few hours later and it is expired.
Using a new account to post this.

**Comment by donerre2** [comment]
same with my acc @donerre since yesterday morning. i had to create this one to open a ticket

**Reply by donerre** [reply]
after +- 24hrs trying i just got a code 3 mins before it would expire.. dont wanna give hope, but maybe u should try again ppl!

**Comment by Bahriddin-Boboyev** [comment]
I'm having this problem too :(

**Comment by Dawid-Witkowski** [comment]
Having the same issue.

**Comment by PatrickTsu** [comment]
Same problem with my old account since yesterday morning.

**Comment by canuckjacq** [comment, marked as answer]
We have a public status for this now, please follow [here](https://www.githubstatus.com/incidents/b6f4j9ljz9r6) for updates.

**Comment by ghost** [comment]
I have repeatedly had this issue when accessing Git Hub through my HA trying to integrate hacs. At that level I get stuck at the device verification level because I never received the email containing the 8 digit code to my primary, verified email address.
